CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -161,6 +162,8 @@ void PuppiProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
             pReco.id = 0;
             if ((fPtMaxCharged > 0) and (pReco.pt > fPtMaxCharged))
               pReco.id = 1;
+            else if ((fEtaMaxCharged > 0) and (std::abs(pReco.eta) > fEtaMaxCharged))
```

Comment:
When one wants to say "do nothing", if one sets "EtaMaxCharged" to some large value (instead of 0), this line could be replaced by the more intuitive
```C++
            else if  (std::abs(pReco.eta) > fEtaMaxCharged)
``` 
(which is by the way what is done later on in this same code for "EtaMaxPhotons")

Reply:
done